projects/plugins/jetpack/modules/memberships/class-jetpack-memberships.php

```diff
@@ -467,12 +481,26 @@ public function register_gutenberg_block() {
 			$deprecated = function_exists( 'gutenberg_get_post_from_context' );
 			$uses       = $deprecated ? 'context' : 'uses_context';
 			Blocks::jetpack_register_block(
-				'jetpack/recurring-payments',
+				'jetpack/recurring-payments-button',
 				array(
 					'render_callback' => array( $this, 'render_button' ),
 					$uses             => array( 'isPremiumContentChild' ),
 				)
 			);
+			Blocks::jetpack_register_block(
+				'jetpack/recurring-payments',
+				array(
+					'render_callback' => array( $this, 'render_container' ),
+					$uses             => array( 'isPremiumContentChild' ),
+					'supports'        => array(
```

Comment:
Ideally this block would use a [block.json](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/) to avoid repeating this in the js and the php, but that feels like a PR for another day.